exercises/connect/connect.js

```diff
@@ -0,0 +1,18 @@
+//
+// This is only a SKELETON file for the 'Connect' exercise. It's been provided as a
+// convenience to get you started writing code faster.
+//
+
+export class Board {
+  constructor() {
+    throw new Error("Remove this statement and implement this function");
+  }
+
+  winner() {
+    throw new Error("Remove this statement and implement this function");
+  }
+
+  score() {
```

Comment:
`score` is an internal function not required by [specs](https://github.com/exercism/javascript/blob/d1fcd01c54c213b5a6116de0d9e296028e70b0f1/exercises/connect/connect.spec.js). Users are free to have their own implementation with different internal functions.